src/pydap/parsers/dds.py

```diff
@@ -8,25 +8,22 @@
 from ..model import (DatasetType, BaseType,
                      SequenceType, StructureType,
                      GridType)
-from ..lib import quote, STRING
-
-typemap = {
-    'byte':    np.dtype("B"),
-    'int':     np.dtype(">i4"),
-    'uint':    np.dtype(">u4"),
-    'int16':   np.dtype(">i4"),
-    'uint16':  np.dtype(">u4"),
-    'int32':   np.dtype(">i4"),
-    'uint32':  np.dtype(">u4"),
-    'float32': np.dtype(">f4"),
-    'float64': np.dtype(">f8"),
-    'string':  np.dtype(STRING),  # default is "|S128"
-    'url':     np.dtype(STRING),
-    }
+from ..lib import (quote, LOWER_DAP2_TO_NUMPY_PARSER_TYPEMAP)

```

Comment:
Check your line endings in this file. Looks like you have \r\n (that should just \n). Just run this through dos2unix or something.